src/Template/Element/language_dropdown_angular.ctp

```diff
@@ -0,0 +1,28 @@
+<div class="language-dropdown-container"
+     title="{{selectedLanguage.name}}">
```

Comment:
This title attribute causes two tooltips on the language dropdown in the navigation bar:
![tooltip](https://user-images.githubusercontent.com/6566321/93206144-3dd8f480-f759-11ea-81d3-9803af3f5083.png)

I don't think that the title is necessary here, because it only repeats the current value of the dropdown.

But after removing it there is still a problem because the tooltip for the navigation bar will hide the first entry of the dropdown after selecting it and not moving the mouse. So I would suggest to add `md-direction='top'` to the `md-tooltip` directive in `src/Template/Element/sentences/navigation.ctp`:
https://github.com/Tatoeba/tatoeba2/blob/98269d3af50a4b45e0c75456307bc13cbb5a9a7e/src/Template/Element/sentences/navigation.ctp#L41-L43

Reply:
Good suggestion. I implemented it.